src/components/TableRow/TableRow.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { useState } from 'react'
+import bin from './bin.png'
+
+const TableRow = ({saved, deleteTask}) => {
+
+    const [complete, setComplete] = useState(false)
+
+    const handleChange = () => {
+        setComplete(!complete)
+    }
+
+    
+  return (
+    <tr className={complete ? 'done' : ''} >
+        <td>{saved.category}</td>
+        <td>{saved.task}</td>
+        <td className='center-clm'><button onClick={handleChange} className={complete ? 'check-task checked' : 'check-task' }/></td>
```

Comment:
The conditional check of 'complete' state is so clever and efficient!!! I was blown away! Never assigned classes using conditionals. Already grateful to be in this group!